Source/Provers/SMTLib/SMTLibProcess.cs

```diff
@@ -74,6 +75,10 @@ public SMTLibProcess(SMTLibOptions libOptions, SMTLibSolverOptions options)
 
     private void SolverExited(object sender, EventArgs e)
     {
+      if (options.Verbosity >= 2) {
+        Console.WriteLine($"[SMT-ERR-{{0}}] Solver exited with code {solver.ExitCode}.");
```

Comment:
I'd ask you to avoid calls to `Console.`, but I see this file already contains them ^,^

Reply:
Yeah, we should refactor it to use something more parameterized.